website/src/pages/downloads.md

```diff
@@ -15,10 +15,12 @@ Repo: https://downloads.apache.org/incubator/fluss/helm-chart
 | [Fluss Source Release](https://downloads.apache.org/incubator/fluss/fluss-0.8.0-incubating/fluss-0.8.0-incubating-src.tgz)                                 | [.asc](https://downloads.apache.org/incubator/fluss/fluss-0.8.0-incubating/fluss-0.8.0-incubating-src.tgz.asc) | [.sha512](https://downloads.apache.org/incubator/fluss/fluss-0.8.0-incubating/fluss-0.8.0-incubating-src.tgz.sha512) |
 | [Fluss Helm Chart](https://downloads.apache.org/incubator/fluss/helm-chart/0.8.0-incubating/fluss-0.8.0-incubating.tgz)                                    | [.asc](https://downloads.apache.org/incubator/fluss/helm-chart/0.8.0-incubating/fluss-0.8.0-incubating.tgz.asc) | [.sha512](https://downloads.apache.org/incubator/fluss/helm-chart/0.8.0-incubating/fluss-0.8.0-incubating.tgz.sha512) |
 | [Fluss Tiering Service Jar](https://repo1.maven.org/maven2/org/apache/fluss/fluss-flink-tiering/0.8.0-incubating/fluss-flink-tiering-0.8.0-incubating.jar) | [.asc](https://repo1.maven.org/maven2/org/apache/fluss/fluss-flink-tiering/0.8.0-incubating/fluss-flink-tiering-0.8.0-incubating.jar.asc) | [.sha1](https://repo1.maven.org/maven2/org/apache/fluss/fluss-flink-tiering/0.8.0-incubating/fluss-flink-tiering-0.8.0-incubating.jar.sha1) |
-| [Connector Jar for Flink 2.1](https://repo1.maven.org/maven2/org/apache/fluss/fluss-flink-2.1/0.8.0-incubating/fluss-flink-2.1-0.8.0-incubating.jar)     | [.asc](https://repo1.maven.org/maven2/org/apache/fluss/fluss-flink-2.1/0.8.0-incubating/fluss-flink-2.1-0.8.0-incubating.jar.asc) | [.sha1](https://repo1.maven.org/maven2/org/apache/fluss/fluss-flink-2.1/0.8.0-incubating/fluss-flink-2.1-0.8.0-incubating.jar.sha1) |
```

Comment:
we should keep this?

Reply:
huh I added it this morning, its weird it deleting it.. let me fix